README.md

```diff
@@ -1,46 +1,73 @@
 # .NET SDK for Lucidtech AI Services
 
+## Documentation
+
+[Link to docs](https://docs.lucidtech.ai/dotnet/v1/index.html)
+
+Create documents by using doxygen.
+Download the latest and greatest version of [doxygen](https://github.com/doxygen/doxygen.git).
+```bash
+$ doxygen documentation.conf
+```
+
 ## Installation
 
 ### NuGet
 
-```
-$ nuget install Lucidtech.LAS.Receipt
+```bash
+$ nuget install Lucidtech.Las
 ```
 
 ## Usage
-Instantiate the Lucidtech.LAS.Client with your API key:
 
-``` C#
-using Lucidtech.LAS;
+### Preconditions
 
-// ...
+- Documents must be in upright position
+- Only one receipt or invoice per document is supported
+- Supported file formats are: jpeg, pdf
+- Lucidtech.Las requires compatibility with .NET standard 4.7
 
-string apiKey = "...";
-var client = new Client(apiKey);
-```
+### Quick start
 
-To scan a receipt, call the API either with a URL to an image file:
 ```C#
-string receiptUrl = "...";
-List<Detection> detections = client.ScanReceipt(receiptUrl);
+using Lucidtech.Las;
+
+# Run inference and create prediction on document 
+using namespace Lucidtech.Las;
+ApiClient apiClient = new ApiClient("<api endpoint>");
+Prediction response = apiClient.Predict(documentPath: "document.pdf", modelName: "invoice");
+Console.WriteLine(response.ToJsonString(Formatting.Indented));
+
+# Send feedback to the model.
+var feedback = new List<Dictionary<string, string>>()
+{ 
+    new Dictionary<string, string>(){{"label", "total_amount"},{"value", "54.50"}},
+    new Dictionary<string, string>(){{"label", "purchase_date"},{"value", "2007-07-30"}}
+};
+FeedbackResponse response = apiClient.SendFeedback(documentId: "<documentId>", feedback: feedback);
+Console.WriteLine(response.ToJsonString(Formatting.Indented));
+
+# Revoke consent and deleting all documents associated with consentId.
+ApiClient apiClient = new ApiClient("<endpoint>"); \n
+RevokeResponse response = apiClient.RevokeConsent(consentId: "<consentId>");
+Console.WriteLine(response.ToJsonString(Formatting.Indented));
+ 
 ```
 
-or with a System.IO.Stream object:
+## Contributing
 
-```C#
-FileStream stream = new FileStream(/* ... */); 
-List<Detection> detections = client.ScanReceipt(stream);
+### Prerequisites
+Download the latest and greatest version of [MSBuild](https://github.com/Microsoft/msbuild).
+
```

Comment:
I think it's missing a step here to get things working: installing nuget package manager